activities/recruitment/hiring-process.md

```diff
@@ -0,0 +1,36 @@
+---
+type: Resource
+---
+# Process for recruiting and hiring candidates
+
+## Roles in this process:
+
+* Head of human resources (HRM), responsible for the hiring process 
+* Chief Executive (CEO), responsible for who we hire
+* Chief Operations (COO), responsible for how this person works in the organisation
+* Directors Foundation for Public Code (BOD), responsible for oversight
+ 
+## Announcing job opening:
+
+1. Place job posting(s) on website
+2. Bring job opening(s) under the attention of network of current team (via mail, LinkedIn, social media)
+3. Bring job opening(s) under the attention of the job opening(s) via (specialized) sites and possibly bureaus
+
+## Processing reactions of people expressing interest:
+
+4. Questions about the content of the position: route to CEO for answering
+5. Questions about the procedure and the conditions: route to HRM for answering
+6. Applications are sent to HRM, for processing
+
+## Planned recruitment and hiring process:
+
+7. Current job openings have no closing date and applications are considered on a rolling basis. When we find a suitable candidate, we hire and close the opening. This means applications are considered ‘by themselves’ and not ‘in competition’ with a certain set of other applications.
+8. HRM confirms receipt to candidates and does first ‘sorting out’.
+9. HRM brings possibly suitable candidates to the attention of CEO en supervisor. These 2 make +/- judgement, without consulting each other. If judgement is not equal, a short discussion follows and judgement is made.
+10. HRM notifies candidates (thank you or invitation for interview).
+11. First interview: CEO and HRM, 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organisation (culture, growth, leadership, long/short term)). Half of the interview is spent on presenting PublicCode.net, in the other half, the candidate has a change to present herself/himself. Immediately after the interview both HRM and CEO sore the candidate on the items on the list and determine if the candidate should progress to a second interview.
+12. Second interview: COO (same person as above) with HRM, also partially based on a pre-established list, and with attention to the particularities and matters not discussed in the first interview. Approximately. 80-90% of the time is spent on questioning the candidate. Immediately  after the interview both M and B/C score the candidate, based on the list and other impressions gathered. HRM takes this input to CEO and HRM and CEO determine if the candidate should receive an offer.
```

Comment:
Cut "(same person as above)" as this it not correct
Replace M with HRM, B/C with CEO and COO

Reply:
fixed in #254